ui-v2/src/components/automations/automations-wizard/action-step/action-change-flow-run-state-fields/flow-states.ts

```diff
@@ -0,0 +1,12 @@
+export const FLOW_STATES = {
+	COMPLETED: "Completed",
+	RUNNING: "Running",
+	SCHEDULED: "Scheduled",
+	PENDING: "Pending",
+	FAILED: "Failed",
+	CANCELLED: "Cancelled",
+	CANCELLING: "Cancelling",
+	CRASHED: "Crashed",
+	PAUSED: "Paused",
+} as const;
```

Comment:
This isn't pretty, but it will ensure we get type errors if the set of states changes.
```suggestion
} as const satisfies Record<
	components["schemas"]["StateType"],
	Capitalize<Lowercase<components["schemas"]["StateType"]>>
>;
```